packages/app/src/app/pages/Sandbox/SearchDependencies/dependencies.js

```diff
@@ -32,24 +37,30 @@ export default createGlobalStyle`
   }
 
   .ais-Highlight-highlighted {
-       color: ${props => props.theme['button.hoverBackground']};
+    color: ${props => props.theme['button.hoverBackground']};
```

Comment:
How about using `button.background` currently, the highlighted text isn't visible properly in light theme. And `button.background` goes in contrast with the background. Check the below screenshot:

Light Theme:
![image](https://user-images.githubusercontent.com/22376783/78969708-3d428f00-7b25-11ea-8ef0-e9439039317b.png)

Dark theme:
![image](https://user-images.githubusercontent.com/22376783/78969823-8bf02900-7b25-11ea-9406-cbc9812bb597.png)

Thoughts ?

Reply:
I and @siddharthkp had the same the discussion about the `highlighted` text issue.
And after using `button.background` the `highlighted` are visible now.

Reply:
I will make the changes and push it.

Reply:
👋 

short: this works, :shipit:! 

I'd recommend using <Text variant="muted"> and <Text> to create a difference between highlighted text and normal text.

It's hard to make this modal to match the redesign of the editor without redesigning the parts of the modal with `@codesandbox/components`.

That's not something that should block this improvement though! I'd recommend merging this PR and continue the work in another PR which can be longer term :)